src/core/EnvelopeAndLfoParameters.cpp

```diff
@@ -24,6 +24,8 @@
 
 #include <QDomElement>
 
+#include <iostream>
```

Comment:
When loading from an audio file, exceptions are now handled outside of the ``SampleBuffer`` constructors, and an error message is printed to the console, hence the need for ``iostream``. This is just one step closer to proper exception handling, as leaving ``SampleBuffer`` in an invalid state (i.e., it failed to load so we just put 1 empty frame as a way of bailing) ignores the callers exceptions, since the caller would've expected it to load successfully if no error was to return.